bikeshed/MetadataManager.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, doc):
         self.issues = []
         self.issueTrackerTemplate = None
 
-        self.otherMetadata = defaultdict(list)
+        self.otherMetadata = OrderedDict()
```

Comment:
I have a DefaultOrderedDict class, just `from DefaultOrderedDict import DefaultOrderedDict`.

(I really hate the "if empty" checks; they're a major source of bugs in my experience. defaultdict is a life-saver.)